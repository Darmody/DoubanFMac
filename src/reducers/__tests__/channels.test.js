import reducer from 'reducers/entities'

describe('reducer::channels', () => {
  it('initialize state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot()
  })
})
