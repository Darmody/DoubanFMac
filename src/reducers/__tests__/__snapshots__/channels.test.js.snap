// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducer::channels initialize state 1`] = `Object {}`;
